app/views/repositories/show.html.erb

```diff
@@ -84,6 +84,24 @@
   <div class="toolbar-filter-buttons" style="display:none">
   </div>
 
+  <div class="print-label-modal-container"
+    data-print-url="<%= print_repository_repository_rows_path(repository_id: @repository) %>"
+    data-zebra-progress-url="<%= print_repository_repository_rows_path(repository_id: @repository) %>"
```

Comment:
For zebra path should be `print_zpl_repository_repository_rows_path(repository_id: @repository)`